src/Selenium2Driver.php

```diff
@@ -13,6 +13,7 @@
 use Behat\Mink\Exception\DriverException;
 use Behat\Mink\Selector\Xpath\Escaper;
 use WebDriver\Element;
+use WebDriver\Exception\InvalidRequest;
```

Comment:
The Scrutinizer CI says that this class import isn't used anymore: https://scrutinizer-ci.com/g/minkphp/MinkSelenium2Driver/inspections/89c102e8-c656-4124-b660-6129c7ec2f16/patches?status=new